### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by flo-dup**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines

### CODE REVIEW

File: pages/contributing/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ You can write it manually but Git even has a -s command line option to append th
 $ git commit -s -m 'This is my commit message'
 ```
 
-Note that a check will be performed during the continuous integration, indicating whether or not commits in a Pull Request do not contain a valid `Signed-off-by` line.
+Note that a check will be performed during the continuous integration, indicating whether commits in a Pull Request do not contain a valid `Signed-off-by` line.
```
Comment (colinepiloquet) [inline pages/contributing/index.md:27]: This still sounds odd. I would write : "indicating whether commits in a Pull Request contain a valid "Signed-off-by" line or not"

**Review by colinepiloquet — COMMENTED** [review]
Just one comment, otherwise, it sounds good!

File: pages/contributing/index.md (hunk 1 above)
Comment (colinepiloquet) [reply pages/contributing/index.md:27]: Or use if...

Comment (flo-dup) [reply pages/contributing/index.md:27]: Thanks @colinepiloquet ! The IDE was telling me that "whether" used with "or not" is incorrect except if used in the meaning of "regardless of whether", and it seems more or less the case after a quick research: https://english.stackexchange.com/questions/3382/whether-or-not-vs-whether

I think we could also simplify it by removing the second not: "whether (or if) all commits in a PR contain (...)".

Comment (colinepiloquet) [reply pages/contributing/index.md:27]: It actually makes sense with the "or not", what seemed strange was more the whether and do not afterwards, I wouldn't put a negation

**Review by colinepiloquet — APPROVED** [review]

**Pull request merged**